modules/base/rendering/grids/renderableradialgrid.h

```diff
@@ -85,6 +87,11 @@ class RenderableRadialGrid : public Renderable {
 
     std::vector<GeometryData> _circles;
     GeometryData _lines{GL_LINES};
+
+    // Labels
+    bool _hasLabels = false;
+    properties::BoolProperty _drawLabels;
+    std::unique_ptr<LabelsComponent> _labels = nullptr;
```

Comment:
No need to allocate `nullptr` to it;  the default constructor does the right thing